src_cpp/elfgames/go/common/go_state_ext.cc

```diff
@@ -82,47 +82,52 @@ std::string GoStateExt::dumpSgf(const std::string& filename) const {
 
 void GoStateExt::showFinishInfo(FinishReason reason) const {
   Stone player = _state.nextPlayer();
-  std::cout << _state.showBoard() << std::endl;
+  _logger->info("{}\n", _state.showBoard());
   std::string sgf_record = dumpSgf("");
-  std::cout << sgf_record << std::endl;
+  _logger->info("{}\n", sgf_record);
 
-  std::cout << "[" << _game_idx << ":" << _seq
-            << "] Current request: " << curr_request_.info()
-            << ", used_model: ";
+  _logger->info("[{}:{}] Current request: {}, used_model: ",
+    _game_idx,
+    _seq,
+    curr_request_.info());
   for (const auto& i : using_models_) {
-    std::cout << i << ", ";
+    _logger->info("{}, ", i);
```

Comment:
This will log one model number per line (recall that logger automatically adds newlines). You probably want to join the models into a string and then log that single string in line 89 (after used_model).